rotkehlchen/tasks/assets.py

```diff
@@ -239,41 +323,62 @@ def update_aave_v3_underlying_assets(chains_aggregator: 'ChainsAggregator') -> N
-                        )],
-                        encounter=TokenEncounterInfo(should_notify=False),
-                    )
-                except DeserializationError as e:
-                    log.error(
-                        'Failed to deserialize Aave v3 reserve token address: '
-                        f'{decoded_reserve_token_address} due to {e!s}',
-                    )
-                except NotERC20Conformant as e:
-                    log.error(
-                        f'Failed to add underlying token {underlying_token_address} for '
-                        f'{decoded_reserve_token.identifier} due to {e!s}',
-                    )
+                if (deserialized_reserve := deserialize_evm_address(reserve)) != ZERO_ADDRESS:
+                    if deserialized_reserve in reserve_to_underlying:
+                        continue  # the same token can appear multiple times
+                    reserve_to_underlying[deserialized_reserve] = underlying_token_address
+
+        # check the tokens missing and query their properties
+        missing_tokens = _aave_tokens_are_correct(reserve_to_underlying, node_inquirer.chain_id)
+        try:
+            address_to_properties = _batch_query_properties(node_inquirer, missing_tokens)
+        except RemoteError as e:
+            log.error(f'Failed to query Aave v3 reserve tokens addresses due to {e!s}')
+            continue
+
+        # add the mappings of underlying tokens to the globaldb
+        encounter = TokenEncounterInfo(should_notify=False)
+        for reserve_address, underlying_address in reserve_to_underlying.items():
+            if (properties := address_to_properties.get(reserve_address)) is None:
+                name = symbol = decimals = None
+            else:
+                name, symbol, decimals = properties
+
+            try:
+                # calling get_or_create_evm_token along with the underlying_tokens attribute
+                # will first ensure that both underlying and reserve tokens info is added,
+                # then the mapping is saved. This order is necessary due to its FK relationship
+                # Here we provide name, symbol and decimals since we have queried them previously
+                get_or_create_evm_token(
+                    userdb=node_inquirer.database,
+                    evm_address=reserve_address,
+                    chain_id=chain_id,
+                    protocol=CPT_AAVE_V3,
+                    token_kind=EvmTokenKind.ERC20,
+                    underlying_tokens=[UnderlyingToken(address=underlying_address, token_kind=EvmTokenKind.ERC20, weight=ONE)],  # noqa: E501
+                    symbol=symbol,
+                    name=name,
+                    decimals=decimals,
+                    encounter=encounter,
+                )
+            except DeserializationError as e:
+                log.error(
+                    'Failed to deserialize Aave v3 reserve token address: '
```

Comment:
shouldn't you also add chain name in both this and the other error log beneat this one?